JPS_Ontology/ontology/ontosealevel/OntoSeaLevel.csv

```diff
@@ -0,0 +1,34 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoSeaLevel,TBox,https://www.theworldavatar.com/kg/ontosealevel/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoSeaLevel,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoSeaLevel,TBox,The Ontology for Sea Level is developed to represent information about sea level.,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoSeaLevel,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+SeaLevel,Class,,,,,,Surface level of oceans.,https://www.theworldavatar.com/kg/ontosealevel/,Sea level
+GlobalMeanSeaLevel,Class,SeaLevel,IS-A,,,,Mean surface level of oceans globally.,https://www.theworldavatar.com/kg/ontosealevel/,Global mean sea level
+LocalMeanSeaLevel,Class,SeaLevel,IS-A,,,,Mean surface level of ocean relative to a specific location.,https://www.theworldavatar.com/kg/ontosealevel/,Local mean sea level
+Damage,Class,,,,,,Damage due to rising sea level,https://www.theworldavatar.com/kg/ontosealevel/,Damage
+EconomicDamage,Class,Damage,IS-A,,,,Economic damage due to rising sea level.,https://www.theworldavatar.com/kg/ontosealevel/,Economic damage
+CulturalDamage,Class,Damage,IS-A,,,,Damage caused to cultural sites due to rising sea level.,https://www.theworldavatar.com/kg/ontosealevel/,Cultural damage
+CulturalSite,Class,,,,,,A place or building of cultural importance.,https://www.theworldavatar.com/kg/ontosealevel/,Cultural site
+SeaLevelMeasurement,Class,,,,,,Sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,Sea level measurement
+ActualSeaLevelMeasurement,Class,SeaLevelMeasurement,IS-A,,,,Actual sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,Actual sea level measurement
+ForecastSeaLevelMeasurement,Class,SeaLevelMeasurement,IS-A,,,,Forecasted sea level measurement based on a forecasting model.,https://www.theworldavatar.com/kg/ontosealevel/,Forecast sea level measurement
+ForecastSeaLevelMeasurement,Class,https://www.theworldavatar.com/kg/ontotimeseries/Forecast,IS-A,,,,Forecasted sea level measurement based on a forecasting model.,https://www.theworldavatar.com/kg/ontosealevel/,
+Tree,Class,CulturalSite,IS-A,,,,Tree,https://www.theworldavatar.com/kg/ontosealevel/,Tree
+HeritageTree,Class,Tree,IS-A,,,,A tree that is the natural heritage of a specific place.,https://www.theworldavatar.com/kg/ontosealevel/,HeritageTree
+https://schema.org/Museum,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Museum
+https://schema.org/TouristAttraction,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Tourist attraction
+https://schema.org/LandmarksOrHistoricalBuildings,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Landmarks or historical building
```

Comment:
Redefining other people's concepts is something that should be avoided, I believe.

Reply:
Created our own class, and gave `equivalentTo` definiton where needed.